tests/test_clickhouse.py

```diff
@@ -36,7 +35,7 @@ def test_reconnect(self, FakeClient):
             errors.NetworkError,
             '{"data": "to my face"}',
         ]
-        cp = ClickhousePool(settings.CLICKHOUSE_HOST, settings.CLICKHOUSE_PORT)
+        cp = ClickhousePool("localhost", 9000)
```

Comment:
These settings are supposed to be valid for the test to work, is there any reason not pick up a host from the cluster configuration? Or to provide a specific single node cluster configuration in settings_test ?

Reply:
These values don't actually need to be valid (they just happen to be the correct host and port in this case). We're mocking the return value so it wouldn't make a difference if there was any other values in here.

Reply:
It would probably be good to make these obviously invalid values so that someone who comes along and reads this later doesn't end up misinterpreting their validity.

In general it seems like these tests know a bit too much about the internals of the objects they are testing but that's probably a different topic out of the scope of this change.